README.md

```diff
@@ -59,14 +59,9 @@ and health-checking mechanisms.
   sudo yum -y install consul-1.9.1
   ```
 
-* Install Motr.
+* Build and Install Motr.
 
-  * .. from RPMs
-    ```sh
-    sudo yum -y install cortx-motr cortx-motr-devel
-    ```
-
-  * .. or from [sources](https://github.com/Seagate/cortx-motr/blob/main/doc/Quick-Start-Guide.rst) use this link to build Motr from source. Once you built Motr continue with the steps listed below.
+  *  from sources use [this](https://github.com/Seagate/cortx-motr/blob/main/doc/Quick-Start-Guide.rst) link to build Motr from source. Once you built Motr continue with the steps listed below.
```

Comment:
Last sentence doesn't look good to me.
Suggestion: Please make sure that Motr is built and then proceed with the steps below.

Reply:
Resolved these comments as part of https://github.com/Seagate/cortx-hare/pull/1610 .